service/src/main/resources/templates/immunization_observation_statement_template.mustache

```diff
@@ -12,5 +12,6 @@
         <text>{{pertinentInformation}}</text>
     </pertinentAnnotation>
 </pertinentInformation>{{/pertinentInformation}}
+        {{{referenceRange}}}
```

Comment:
I think this {{{referenceRange}}} still needs to be removed. It is not part of Immunization